src/authenticationinapp/authenticationinapplistener.h

```diff
@@ -18,8 +18,8 @@ class AuthenticationInAppListener final : public AuthenticationListener {
   explicit AuthenticationInAppListener(QObject* parent);
   ~AuthenticationInAppListener();
 
-  void start(const QString& codeChallenge,
-             const QString& codeChallengeMethod) override;
+  void start(const QString& codeChallenge, const QString& codeChallengeMethod,
+             const QString& emailAddress) override;
```

Comment:
If the `emailAddress` is optional, it might be reasonable to give it a default value.